api/datadoghq/common/datadogpodautoscaler_types.go

```diff
@@ -115,72 +115,72 @@ type DatadogPodAutoscalerScalingRule struct {
-// For pod-level targets, resources are the sum of all containers resources.
+// DatadogPodAutoscalerPodResourceObjective defines a pod-level resource objective (for instance, CPU Utilization at 80%)
+// For pod-level objectives, resources are the sum of all containers resources.
 // Utilization is computed from sum(usage) / sum(requests).
 // +kubebuilder:object:generate=true
-type DatadogPodAutoscalerResourceTarget struct {
+type DatadogPodAutoscalerPodResourceObjective struct {
 	// Name is the name of the resource.
 	// +kubebuilder:validation:Enum:=cpu
 	Name corev1.ResourceName `json:"name"`
 
-	// Value is the value of the target.
-	Value DatadogPodAutoscalerTargetValue `json:"value"`
+	// Value is the value of the objective.
+	Value DatadogPodAutoscalerObjectiveValue `json:"value"`
 }
 
-// DatadogPodAutoscalerContainerResourceTarget defines a container-level resource target (for instance, CPU Utilization for container named "foo" at 80%)
+// DatadogPodAutoscalerContainerResourceObjective defines a container-level resource objective (for instance, CPU Utilization for container named "foo" at 80%)
 // +kubebuilder:object:generate=true
-type DatadogPodAutoscalerContainerResourceTarget struct {
+type DatadogPodAutoscalerContainerResourceObjective struct {
 	// Name is the name of the resource.
 	// +kubebuilder:validation:Enum:=cpu
 	Name corev1.ResourceName `json:"name"`
 
 	// Value is the value of the target.
-	Value DatadogPodAutoscalerTargetValue `json:"value"`
+	Value DatadogPodAutoscalerObjectiveValue `json:"value"`
 
 	// Container is the name of the container.
 	Container string `json:"container"`
 }
 
-// DatadogPodAutoscalerTargetValueType specifies the type of metric being targeted, and should be either
+// DatadogPodAutoscalerObjectiveValueType specifies the type of objective value.
 // kubebuilder:validation:Enum:=Utilization
-type DatadogPodAutoscalerTargetValueType string
+type DatadogPodAutoscalerObjectiveValueType string
 
 const (
-	// DatadogPodAutoscalerUtilizationTargetValueType declares a MetricTarget is an AverageUtilization value
-	DatadogPodAutoscalerUtilizationTargetValueType DatadogPodAutoscalerTargetValueType = "Utilization"
+	// DatadogPodAutoscalerUtilizationObjectiveValueType declares an objective based on a Utilization (percentage, 0-100).
+	DatadogPodAutoscalerUtilizationObjectiveValueType DatadogPodAutoscalerObjectiveValueType = "Utilization"
 )
 
-// DatadogPodAutoscalerTargetValue defines the value of the target.
+// DatadogPodAutoscalerObjectiveValue defines the target value of the objective.
 // +kubebuilder:object:generate=true
-type DatadogPodAutoscalerTargetValue struct {
-	// Type specifies how the value is expressed (Absolute or Utilization).
-	Type DatadogPodAutoscalerTargetValueType `json:"type"`
+type DatadogPodAutoscalerObjectiveValue struct {
+	// Type specifies how the value is expressed (Utilization only at the moment).
```

Comment:
```suggestion
	// Type specifies how the value is expressed (Utilization only).
```

Just making this suggestion because the docs team prefers not having references to time